### ISSUE TITLE
A bug when training the model

### ISSUE THREAD

**Issue by taoxinhao13** [rockeyben/DCCF#8]
Hello, this is an interesting job. But when I run the script "./runs/train_dccf_idih_hrnet18s_v2p.sh" to train, I found there's some bugs that the loss is "nan". How can I deal with it?
![image](https://user-images.githubusercontent.com/56617678/220900548-2106faec-9814-4914-a752-07ef83683503.png)

**Comment by rockeyben** [comment]
It seems that the high resolution loss is 0 in your log, and total step is 4108, so I assume you are training the low resolution part with batchsize=16. I run the script according to your setting and I get the following log, the loss is converging normally:
<img width="1440" alt="截屏2023-02-24 14 56 51" src="https://user-images.githubusercontent.com/27840904/221112843-4f76bde5-448b-466e-821b-c172df1da8d5.png">
Maybe you should check something like:
  1. pytorch versions or other enviroments
  2. your initial weight is 'pretrained_models/hrnet_w18_small_model_v2.pth'
  3. you haven't modified configs in 'iharm/mconfigs/' or 'models/fixed256/'
  4. data paths

**Comment by taoxinhao13** [comment]
Hello, I found that if I use l,s,h,rgb loss, the bug will appear. What's more, I found whether i chose the learning rate of these losses, the loss will like this (0.12->0.01->0.005->3->nan),and there's very confusion problem, maybe the cuda version? Well, I think I can finetune the model that fdidn't use these losses. Thank you very much.

**Comment by taoxinhao13** [comment]
Hello, I found that if I use l,s,h,rgb loss, the bug will appear. What's more, I found whether I chose the learning rate of these losses, the loss will like this (0.12->0.01->0.005->3->nan),and there's very confusion problem, maybe the cuda version? Well, I think I can finetune the model that fdidn't use these losses. Thank you very much.

**Comment by rockeyben** [comment]
It seems that you start training the model with an initial weights, what is it? If you use some weights with suffix 'notv', you should use config files with suffix 'sat_clamp' or set 'clamp_sat_modify' =True in ' [iharm/mconfigs/base.py](https://github.com/rockeyben/DCCF/blob/master/iharm/mconfigs/base.py).

Otherwise this can be a cuda problem, my cuda version is 10.2.

**Comment by taoxinhao13** [comment]
Well, that doesn't work, maybe because my cuda version is 11.7 so I have to use new version pytorch. Actually, we are finetuning the models to a smaller dataset, so remove some losses to fix some paramters is also OK. Thanks a lot for your detailed reply, I learnt a lot and I'll star this work :)
